backoffice/src/Models/ViewModels/EmailView.cs

```diff
@@ -16,11 +16,13 @@ public EmailView(Email email)
         TypeOfEmail = email.GetType();
         Date = email.DateLastModified;
         Subject = email.Subject;
+        SentDate = email.DateSent;
     }
     
     public Guid Id { get; set; }
     public string? Recipient { get; set; }
     public Type TypeOfEmail { get; set; }
     public DateTime Date { get; set; }
     public string? Subject { get; set; }
+    public DateTime SentDate { get; set; }
```

Comment:
SentDate should be type DateTime? This is a nullable date. As the starting value of this will always be null until explicitly sent

You'll then be able to have 

public bool IsSent => SentDate.HasValue;

 in the EmailView model

Reply:
done